das_client/lib/app/pages/journey/train_journey/widgets/table/service_point_row.dart

```diff
@@ -36,6 +36,11 @@ class ServicePointRow extends BaseRowBuilder<ServicePoint> {
     );
   }
 
+  @override
```

Comment:
wozu dieser override?

Reply:
Die `timeCell` ist nicht mehr Teil von der `BaseRow`. Macht auch mehr Sinn in der `ServicePointRow`, da die anderen Elemente keine Zeit haben.